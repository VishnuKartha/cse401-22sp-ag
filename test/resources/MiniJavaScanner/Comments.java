// Test a Comment
int i = 0;
/*
    Much
    Longer
    Comment
*/
int j = 0;