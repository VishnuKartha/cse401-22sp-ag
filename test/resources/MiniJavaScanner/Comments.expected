INT ID(i) BECOMES INTEGER_LITERAL(0) SEMICOLON INT ID(j) BECOMES INTEGER_LITERAL(0) SEMICOLON